docs/connector-development/tutorials/building-a-python-source.md

```diff
@@ -232,9 +232,9 @@ For a brief overview on the catalog check out [Beginner's Guide to the Airbyte C
 
 As described in the template code, this method takes in the same config object as the previous methods. It also takes in a "configured catalog". This object wraps the catalog emitted by the `discover` step and includes configuration on how the data should be replicated. For a brief overview on the configured catalog check out [Beginner's Guide to the Airbyte Catalog](../../understanding-airbyte/beginners-guide-to-catalog.md). It then returns a generator which returns each record in the stream.
 
-### Step 8: Set up Standard Tests
+### Step 8: Set up Connector Acceptance Tests (CATs)
 
-The Standard Tests are a set of tests that run against all sources. These tests are run in the Airbyte CI to prevent regressions. They also can help you sanity check that your source works as expected. The following [article](../testing-connectors/connector-acceptance-tests-reference.md) explains Standard Tests and how to run them.
+The Connector Acceptance Tests are a set of tests that run against all sources. These tests are run in the Airbyte CI to prevent regressions. They also can help you sanity check that your source works as expected. The following [article](../testing-connectors/connector-acceptance-tests-reference.md) explains Connector Acceptance Tests and how to run them.
 
 You can run the tests using `./gradlew :airbyte-integrations:connectors:source-<source-name>:integrationTest`. Make sure to run this command from the Airbyte repository root.
```

Comment:
This doesn't work anymore it need to update to use the `airbyte-ci` cli instead, do you mind changing it?

Reply:
Yes! I'll get to this — there are multiple spots where this script is still mentioned, and I'll clean it up too.

Reply:
Oh wait, I've already cleaned that one up 🙃